assignmentTest/splab/Gopistol/invitation/src/main/java/com/splab/invitation/domain/member/service/EmailService.java

```diff
@@ -0,0 +1,67 @@
+package com.splab.invitation.domain.member.service;
+
+import com.splab.invitation.domain.member.entity.Member;
+import com.splab.invitation.domain.member.service.dto.InviteMemberCommand;
+import com.splab.invitation.persistence.member.MemberRepository;
+import jakarta.mail.Message.RecipientType;
+import jakarta.mail.internet.InternetAddress;
+import jakarta.mail.internet.MimeMessage;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class EmailService {
+
+  private final MemberRepository repository;
+  private final JavaMailSender mailSender;
+  private final ResourceLoader resourceLoader;
+  private static final String prefix = "http://localhost:8080/members/joinMember?";
+  private static final String CONTENT_PLACEHOLDER = "CONTENT";
+  private static final String LINK_PLACEHOLDER = "YOUR_INVITATION_LINK";
+
+  public void sendEmail(InviteMemberCommand command, String passKey) {
+    // 링크 생성
+    String link = prefix + "email=" + command.getEmail() + "&passKey=" + passKey;
```

Comment:
문자열을 만든다면 foramt을 이용하면 가독성이 높아집니다.
```suggestion
    String link = String.format("%s?email=%s&passkey=%spasskey", prefix, command.getEmail(), passkey)
```

url을 형식이라면 url을 만들어서 stirng으로 변환하는 것도 좋은 방법입니다
```suggestion
    String link = UriBuilder
        .fromUri("http://localhost:8080")
        .path("/members/joinMember")
        .queryParam("email",  command.getEmail())
        .queryParam("passkey",  passkey)
        .build()
        .toString();
```

Reply:
오 가독성에 훨씬 좋은것같습니다